### PR TITLE
Decorator Tweaks

### PR DESCRIPTION
**Pull request by dpendic**
signed-off-by: Jincux dephurites@gmail.com
- Fixed the max height of cacti. No more 4-tall cacti
- Made trees like normal trees (cut the edges)

### CODE REVIEW

**Comment by windy1** [comment]
This needs to be squashed into one commit and signed off please, if you need help please refer to the README

**Comment by dpendic** [comment]
-snip-

Fixed.

File: src/main/java/org/spout/vanilla/generator/normal/decorator/TreeDecorator.java [hunk 1]
```diff
@@ -77,7 +77,8 @@ private void generateSmallTree(Chunk c, Random ra, int cx, int cy, int cz) {
 		for (int k = oneWidth + 1; k <= oneWidth + twoWidth; k++) {
 			for (int i = -2; i <= 2; i++) {
 				for (int j = -2; j <= 2; j++) {
-					w.setBlockMaterial(cx + i, cy + height - k, cz + j, VanillaMaterials.LEAVES, c.getWorld());
+					if(!(j == -2 && i == -2) && !(j == 2 && i == -2) && !(j == -2 && i == 2) && !(j == 2 && i == 2))
```
Comment (Top-Cat) [inline src/main/java/org/spout/vanilla/generator/normal/decorator/TreeDecorator.java]: Use the one true brace style. Space after if, braces surrounding even single-line conditionals.

**Comment by dpendic** [comment]
Fixed. Sorry, first time contributing to any of the Spout projects. Won't happen again.

**Pull request merged**